pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -188,9 +190,15 @@ func getVirtualMachineByName(cfg *VSphereConfig, ctx context.Context, c *govmomi
 	}
 	f.SetDatacenter(dc)
 
+	var workingDir string
+	if len(cfg.Global.WorkingDir) > 0 {
+		workingDir = path.Clean(cfg.Global.WorkingDir) + "/"
```

Comment:
The initialization code should add '/' when it's missing from the user-configured working-dir value.  This would standardize the input and avoid a double '/' case.